crates/blob-explorers/src/response.rs

```diff
@@ -163,10 +173,6 @@ impl TransactionDetails {
 #[derive(Default, Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct BlockInfo {
-    /// Block hash.
-    pub hash: B256,
-    /// Block timestamp.
-    pub timestamp: chrono::DateTime<chrono::offset::Utc>,
```

Comment:
aren't those still available?
feels odd that those are removed

<img width="274" alt="image" src="https://github.com/user-attachments/assets/a279cdd7-556a-4a5f-9784-532b398d2095">

Reply:
These fields are present when fetching a block, for which `BlockResponse<T>` has these fields—`BlockInfo` doesn't need them.